docs/preparing-your-app-for-testflight.md

```diff
@@ -0,0 +1,143 @@
+# Preparing your App for TestFlight
+
+Publishing to the App Store requires some preparation. This guide should help you get started with using Transporter and the App Connect site to upload your app to the App Store for internal testing via TestFlight. This guide will use the example of publishing a MacCatalyst app to the Mac App Store, but the process is similar for iOS and tvOS apps.
+
+There is some preparation involved with preparing your Apple Developer account for publishing, such as preparing an App Identifier and a Provisioning Profile. The MAUI documentation has a great guide for that:
+https://aka.ms/maui-publish-app-store
+
+Also, your project will need to have configuration values set in the .csproj file, the Info.plist file, and the Entitlements.plist file.
+
+## Preparing your Project File (.csproj) for App Store submission
+
+Before you can submit your app to the App Store, you need to make sure that it is correctly configured. This includes:
+
```

Comment:
Each of these should have tell where and how to set these and or an explanation that answers the question "how do I know it's correct?"

Maybe insert an annotated sample .csproj with each of these items set to a correct value?